clang/test/CodeGenSYCL/kernelname-enum.cpp

```diff
@@ -155,3 +182,5 @@ int main() {
 // CHECK: template <> struct KernelInfo<::dummy_functor_6<(unscoped_enum)0>>
 // CHECK: template <> struct KernelInfo<::dummy_functor_7<::no_namespace_int>>
 // CHECK: template <> struct KernelInfo<::dummy_functor_7<::internal::namespace_short>>
+// CHECK: template <> struct KernelInfo<::T1<::T2<(type_argument_template_enum::E)0>>>
```

Comment:
This inconsistency is fixed in the followup patch, right?

Reply:
Yes.